ansible-ipi-install/README.md

```diff
@@ -418,6 +419,53 @@ With the `playbook.yml` set and in-place, run the `playbook.yml`
 $ ansible-playbook -i inventory/hosts playbook.yml
 ~~~
 
+## Disconnected Registry
+A disconnected registry can be used to deploy the cluster.
+This registry can exist or can be created.
+
+To use a disconnected registry, set the registries host name in 
+the `[registry_host]` group in the inventory file.
+
+### Using an Existing Disconnected Registry
+The `disconnected_registry_auths_file` and the 
+`disconnected_registry_mirrors_file` variables must be set.
+
+The `disconnected_registry_auths_file` variable should be json data. This will be appended to the `auths` section of the pull secret.
+
+The `disconnected_registry_mirrors_file` variable should contain the `additionalTrustBundle` and `imageContentSources` for the disconnected registry.
+~~~
+additionalTrustBundle: |
+  -----BEGIN CERTIFICATE-----
+  The servers certificate should go here
+  -----END CERTIFICATE-----
+  
+imageContentSources:
+- mirrors:
+  - disconnected.example.com:5000/ocp4/openshift4
+  source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+- mirrors:
+  - disconnected.example.com:5000/ocp4/openshift4
+  source: registry.svc.ci.openshift.org/ocp/release
```

Comment:
These details should be in the inventory/hosts.sample file of the README.md and include comments above them (see existing examples). If the variables are part of the install-config.yaml pls put them in the `Vars regarding install-config.yaml` section of the inventory/hosts.sample file

Reply:
Would it be a little confusing to the reader if this is put there before its discussed?

Reply:
done in next commit